apps/dot-voting/app/Decisions.js

```diff
@@ -19,7 +19,7 @@ const EmptyIcon = () => <img src={emptyIcon} alt="" />
 
 class Decisions extends React.Component {
   static propTypes = {
-    app: PropTypes.object.isRequired,
+    app: PropTypes.object,
```

Comment:
Still required?

Reply:
No. The easiest thing to do now - because it is valid for the app to not be there initially - and because we probably don't want to make it overcomplicated with multi-level PropTypes (i.e., it can be `null`, it can be `object` - is to make it not required and just check that if it's present, it's an object.